interface/themes/style_tan.css

```diff
@@ -655,6 +655,8 @@ tr.odd {
   border-radius: 5px;
   font-size:0.8em;
   box-shadow: 2px 1px 4px #88888B;
+  width: 80px;
+  text-align: center;
```

Comment:
Try to narrow down your fix as much as possible to the affected file to avoid breaking styles or code in other modules, modifying this theme file like this might cause  a global effect in codebase. create a style class for that particular use case a apply the style other than applying is to all button input types. Please try not to do it here except you are going to create another class name.